source/read/count.txt

```diff
@@ -0,0 +1,145 @@
+.. uses tutorial.rst
+
+.. _pymongo-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation
+
+Overview
+---------
+
+In this guide, you can learn how to get an accurate and estimated count of the
+number of documents in your collection.
+
+Get an Accurate Count
+----------------------
+
+Use the ``count_documents()`` command to count the number of documents that are in a
+collection. To count the number of documents that match a specific search
+critera, pass the ``count_documents()`` method a dictionary that specifies a
+query.
+
+To learn more about specifying a query, see :ref:`pymongo-specify-query`.
+
+Count All Documents
+~~~~~~~~~~~~~~~~~~~
+
+To return a count of all documents in the collection, pass an empty dictionary to
+the ``count_documents()`` method, as shown in the following example:
+
+.. code-block:: python
+
+   collection = database["collection_name"]
+
+   collection.count_documents({})
+
+Count Specific Documents
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To return a count of documents that match specific search criteria, specify your
+query in the ``count_documents()`` method, as shown in the following example:
+
+.. code-block:: python
+
+   collection = database["collection_name"]
+   
+   collection.count_documents({ "author": "Mike" })
+
+Customize Count Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``count_documents()`` method optionally accepts additional
+parameters, which represent options you can use to configure the count
+operation. If you don't specify any additional options, the driver does not customize
+the count operation.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``comment``
+     - | A comment to attach to the operation.
+
+   * - ``session``
+     - | An instance of ``ClientSession``.
+
+   * - ``skip``
+     - | The number of documents to skip before returning results.
+
+   * - ``limit``
+     - | The maximum number of documents to count. Must be a positive integer.
+
+   * - ``maxTimeMS``
+     - | The maximum amount of time to allow the operation to run, in
+         milliseconds.
+     
+   * - ``collation``
+     - | An instance of ``Collation``.
+
+   * - ``hint``
+     - | Gets or sets the index to scan for documents.
+
+Get an Estimated Count
```

Comment:
```suggestion
Estimate Number of Documents
```

Reply:
I left this (but changed "get" to "retrieve") so that it mirrors the wording in the Overview section and in the other main header.